### PR TITLE
Remove unused variable and struct field

### PR DESCRIPTION
**Pull request by superbrothers**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

In staging/src/k8s.io/kubectl/pkg/cmd/config/view.go,
`defaultOutputFormat` variable is not used. In addition, `OutputFormat`
field of `ViewOptions` struct is not also used.

**Which issue(s) this PR fixes**:

None

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/config/view.go [hunk 1]
```diff
@@ -44,8 +44,7 @@ type ViewOptions struct {
 	Minify       bool
 	RawByteData  bool
 
-	Context      string
-	OutputFormat string
+	Context string
```
Comment (drsantos20) [inline staging/src/k8s.io/kubectl/pkg/cmd/config/view.go]: I would recommend keeping the same indentation space like the previous variable

**Review by drsantos20 — CHANGES_REQUESTED** [review]

File: staging/src/k8s.io/kubectl/pkg/cmd/config/view.go (hunk 1 above)
Comment (superbrothers) [reply staging/src/k8s.io/kubectl/pkg/cmd/config/view.go]: This is the result of running `go fmt`.

**Pull request closed without merging**